products/customer_analytics/frontend/components/ZendeskTicketsFilters/Status.tsx

```diff
@@ -0,0 +1,34 @@
+import { useActions, useValues } from 'kea'
+
+import { LemonSelect } from '@posthog/lemon-ui'
+
+import { capitalizeFirstLetter } from 'lib/utils'
+
+import { zendeskTicketsFiltersLogic } from './zendeskTicketsFiltersLogic'
+
+export const StatusFilter = (): JSX.Element => {
+    const { status } = useValues(zendeskTicketsFiltersLogic)
+    const { setStatus } = useActions(zendeskTicketsFiltersLogic)
+
+    const options = ['all', 'new', 'open', 'hold', 'pending', 'solved', 'closed', 'deleted']
+
+    const label = (key: string) => {
+        switch (key) {
+            case 'all':
+            case null:
+                return 'All statuses'
+            default:
+                return capitalizeFirstLetter(key)
+        }
+    }
```

Comment:
(same comment as before) these could be outside the component to avoid re-creating the function every render